README.md

```diff
@@ -50,17 +52,21 @@ Several template variables are made available to the script running inside the t
 * `title` subject (%s)
 * `messageLines` array of body lines (%b)
 
-`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for usage example.
+
+`options` the object documented below. Useful for parsing the repository name. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+
+`request` is an instance of [sync-request](https://www.npmjs.com/package/sync-request). This can be useful for querying the Jira API for example to extract extra metadata about a ticket related to a commit.
```

Comment:
I'm not comfortable having a `request` library in the templating layer. Ideally templating is a pure function of the input. If anything the logic to get additional data should be in the parsing / data generation phase.

I understand your use case, but I'm wondering if it should live in this library, things can get really complex very soon.
What I'd do here is

```
git-release-notes a..b json | your-script-to-get-stuff-from-jira | git-release-notes html
```

(1) use this tool the get a list of commits in a json format or whatever is handy for you
(2) do your processing with requests and whatever else you need to do to modify your history
(3) use this tool again to generate a template given a json input

Note that (1) we don't currently have a json output but is should be trivial, and (3) take a bit more work so if we get stuff from stdin we don't look at the git history but work with what's given...

What do you think?